src/pages/BodyWeight/views/AddWeight/views/Form/Form.tsx

```diff
@@ -21,11 +21,11 @@ export const Form: React.FC = () => {
   } = useBodyWeightUpdate();
   const { handleSubmit } = methods;
 
+  // snackbar
   const snackbar = useSnackbar();
-
   useEffect(() => {
     if (updateWeightError) {
-      snackbar("Something goes wrong. Please try later.", SnackbarStatus.ERROR);
+      snackbar(updateWeightError.message, SnackbarStatus.ERROR);
```

Comment:
te snackbary chyba też powinny być w miejscu gdzie robimy request, nie zwróciłem uwagi na to wcześniej